src/GafferImage/OpenColorIOTransform.cpp

```diff
@@ -112,6 +130,36 @@ OpenColorIO::ConstContextRcPtr OpenColorIOTransform::getLocalContext(OpenColorIO
 {
 
 	OpenColorIO::ConstContextRcPtr context = config->getCurrentContext();
+	const CompoundDataPlug *p = contextPlug();
+	if(!p)
+	{
+		return context;
+	}
+
+	if( !p->children().size() )
+	{
+		return context;
+	}
+
+	OpenColorIO::ContextRcPtr mutableContext;
+	std::string name;
+	std::string value;
+
+	for( CompoundDataPlug::MemberPlugIterator it( p ); !it.done(); ++it )
+	{
+		IECore::DataPtr d = p->memberDataAndName( it->get(), name );
+		if( d )
+		{
+			value = runTimeCast<StringData>(d)->readable();
```

Comment:
of course this crash if the data type of the plug is anything else than string, but if you succeed to block any other type for the UI, would that be ok?